src/android/Text.tsx

```diff
@@ -40,6 +40,7 @@ export class Text extends CommonText {
                 importantForAccessibility={ importantForAccessibility }
                 numberOfLines={ this.props.numberOfLines === 0 ? null : this.props.numberOfLines }
                 allowFontScaling={ this.props.allowFontScaling }
+                maxFontSizeMultiplier={ this.props.maxFontSizeMultiplier }
```

Comment:
I think we need this to be able to override the default in specific instances. For example, we may want the `maxContentSizeMultiplier` to be much higher for text inside message bubbles vs the text in the navigational elements.